<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkstatus</class>
 <widget class="QWidget" name="networkstatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>493</height>
   </rect>
  </property>
  <widget class="QLabel" name="labelTotalConnections">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total connections:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartupTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Since startup:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMessageCount">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed 0 person-to-person messages.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPubkeyCount">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed 0 public keys.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBroadcastCount">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed 0 broadcasts.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLookupsPerSecond">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory lookups per second: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBytesRecvCount">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Down: 0 KB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBytesSentCount">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Up: 0 KB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSyncStatus">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Objects to be synced:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="STableWidget" name="tableWidgetConnectionCount">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>212</red>
            <green>208</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>212</red>
            <green>208</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>212</red>
            <green>208</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Stream #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Connections</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>STableWidget</class>
   <extends>QTableWidget</extends>
   <header>bitmessageqt/settingsmixin.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bitmessage_icons.qrc"/>
 </resources>
 <connections/>
</ui>
